submodules/storage/variables.tf

```diff
@@ -116,3 +116,11 @@ variable "network_info" {
     })
   })
 }
+
+variable "irsa" {
+  description = "IRSA object."
```

Comment:
Can we make this more descriptive? I realize someone setting up IRSA already knows, but it would be good to save people who don't a search for definitions or docs.